django_nitro_mailer/tests/test_tasks.py

```diff
@@ -0,0 +1,169 @@
+import pytest
+import pickle
+from unittest.mock import patch, MagicMock
+from django.conf import settings
+from django.core.mail import EmailMessage
+from django_nitro_mailer.tasks import send_emails
+from django_nitro_mailer.models import Email, EmailLog
+from unittest.mock import patch
+
+
+@pytest.mark.django_db
+def test_set_and_get_email():
+    email_instance = Email.objects.create(email_data=b"")
+
+    email_message = EmailMessage(
+        subject="Test Subject", body="Test Body", from_email="from@example.com", to=["to@example.com"]
+    )
+
+    email_instance.set_email(email_message)
+
+    assert email_instance.email.subject == "Test Subject"
+    assert email_instance.email.body == "Test Body"
+    assert email_instance.recipients == ["to@example.com"]
+
+
+@pytest.mark.django_db
+def test_email_model():
+
+    email_instance = Email.objects.create(priority=Email.Priorities.HIGH, email_data=b"")
+    assert email_instance.priority == Email.Priorities.HIGH
```

Comment:
Ten test raczej zbędny, bo to jest zwykłe pole, więc na pewno będzie działać (nie testujemy podstawowej funkcjonalności Django)